### ISSUE TITLE
Circle ignoring StrokeColor and StrokeWidth?

### ISSUE THREAD

**Issue by eltomek** [Mapsui/Mapsui#1100]
The following code generates transparent circles but the `StrokeColor` and `StrokeWidth` properties seem to be ignored:
```
Circle circle = new Circle
{
    Center = waypointNode.Value.Position,
    Radius = Distance.FromKilometers(1),
    Quality = 60,
    FillColor = Xamarin.Forms.Color.Transparent,
    StrokeColor = Xamarin.Forms.Color.Yellow,
    StrokeWidth = 10
};
mapView.Drawables.Add(circle);
```
![image](https://user-images.githubusercontent.com/3283040/111368052-ae73b600-8695-11eb-92a9-ad07d998c94f.png)

No matter what `StrokeColor` I assign (even the `Xamarin.Forms.Color.Transparent`) and `StrokeWidth` value the circle is always drawn with 1px thick gray color. `FillColor` works ok.

**Comment by charlenni** [comment]
This was a bug. Should be solved by PR #1101.

**Comment by pauldendulk** [comment]
#1101 Was merged.
